### PR TITLE
[Changelog] Add SE-0242

### PR DESCRIPTION
**Pull request by Azoy**
cc: @slavapestov

### CODE REVIEW

**Comment by slavapestov** [comment]
@swift-ci Please smoke test

**Comment by slavapestov** [comment]
I'll do the 5.1 cherry pick since I need to add an entry of my own. Thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -7511,6 +7527,7 @@ Swift 1.0
 [SE-0228]: <https://github.com/apple/swift-evolution/blob/master/proposals/0228-fix-expressiblebystringinterpolation.md>
 [SE-0230]: <https://github.com/apple/swift-evolution/blob/master/proposals/0230-flatten-optional-try.md>
 [SE-0235]: <https://github.com/apple/swift-evolution/blob/master/proposals/0235-add-result.md>
+[SE-0242]: <https://github.com/apple/swift-evolution/blob/master/proposals/0242-default-values-memberwise-md>
```
Comment (benrimmington) [inline CHANGELOG.md]:
```suggestion
[SE-0242]: <https://github.com/apple/swift-evolution/blob/master/proposals/0242-default-values-memberwise.md>
```

**Comment by slavapestov** [comment]
@swift-ci Please smoke test

**Pull request merged**